src/main/welcomeview/welcomeview.ts

```diff
@@ -723,7 +717,6 @@ export class WelcomeView {
           this.showNotification(installMessage, false);
         } else {
           this.enableLocalServerActions(true);
-          this.showNotification('', false);
```

Comment:
I think this code was always clearing out the success notification. Is there any reason you think we need to keep this around?